docs/delivering-your-session.md

```diff
@@ -0,0 +1,39 @@
+# Delivering your session
+
+It's time to go live! Deep breath - you've got this!
+
+Couple of last notes to keep in mind
+
+## Getting started
+
+- Begin your stream five minutes before your scheduled start time with the **Starting soon** scene
+  - Make sure your mic is muted :-D
+- Start with introductions
+  - Introduce yourself and any co-hosts
+    - Try to limit the amount of time spent on introductions
+    - People are there to see you in action!!
+  - Introduce your topic and target audience
+    - Highlight your outline
+    - Set pre-requisites for the audience
+    - Consider touching on topics you won't cover
+
+### During the session
+
+- Answer any questions you feel comfortable based on your [questions plan](./preparing-your-session.md#Preparing-for-questions)
```

Comment:
I don't think the link you listed exists.

I think it's ./preparing-your-session.md#Plan-for-questions